test/fin_cases.lean

```diff
@@ -1,5 +1,16 @@
 import tactic.fin_cases
+import data.nat.prime
+import group_theory.perm
+import tactic.norm_num
 
+example (p : ℕ) (h : p ∈ list.range 2) : true :=
```

Comment:
This test looks unrelated to `fin_cases`, is it here intentionally?

Reply:
Sorry, I missed that in cleanup. It's gone now.